solutions/206.py

```diff
@@ -0,0 +1,32 @@
+"""
+Find the unique positive integer whose square has the form 1_2_3_4_5_6_7_8_9_0,
+where each “_” is a single digit.
+Since last digit is 0,2nd last will also be 0
+Now we are left with 8 blanks with a total possiblity of 10^8 numbers lets check which is perfect square
+"""
+from math import sqrt
+digits = []
+for digit in range(0,10):
+    digits.append(digit)
+
+number = 0
+for i in range(1,10):
+    number = number * 100 + i
+#number *= 100
+
+def isSquare(n):
```

Comment:
I like this, and the other function that you're using. I'd propose actually moving these into the file `eulib.py` just for consistency (and in case we ever want to use those again. I haven't tested that this gives the solution yet - will do so shortly.

Reply:
Thanks will so at once

Reply:
I have a question - how did you come across this? Are you doing this for #hacktober?

Reply:
I came across this in my summer vacation and solved the problem then only but didn't send a pull request at that time and truth be told I sent it right now due to #hacktoberfest only

Reply:
Also I don't think having singleDigitIterator in a different file would serve purpose of this repository as that would make my method I very difficult to understand specially for new comers. Same goes for checkPossibleSolution and generateAllPossibleSolutions and moreover these two functions are highly subjective to this problem and are unlikely to be reused in some other question. I have put IsSquare function in eulib.py as it can be reused and does not affect understandability of current solution/